src/observe.js

```diff
@@ -1685,7 +1685,8 @@
 
   var expose = global;
 
-  if (typeof exports !== 'undefined') {
+  if (typeof exports !== 'undefined' && exports + '' === '[object Object]' &&
```

Comment:
`exports + '' === '[object Object]'` ... is this equivalent to `Object.getPrototypeOf(exports) === Object.prototype`?